src/SprykerEco/Zed/ComputopApi/Business/Converter/AuthorizeConverter.php

```diff
@@ -13,19 +13,19 @@
 class AuthorizeConverter extends AbstractConverter implements ConverterInterface
 {
     /**
-     * @param array $response
+     * @param array $decryptedResponse
```

Comment:
Can we specify keys and values types in the array? E.g. `array<string, mixed>`
Please check other places.

Reply:
Fixed